### PR TITLE
Ex1.1

### PR DESCRIPTION
**Pull request by Igrass97**
Exercise 1.1

### CODE REVIEW

File: Topic 0/CircleClass/src/TestCircle.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		System.out.println("Area circulo 1 = " + c1.getArea());
+		System.out.println("Area circulo 2 = " + c2.getArea());
+		System.out.println("Area circulo 3 = " + c3.getArea());
+		
+		System.out.println();
+		
+		c3.setColor("Gold");
+		c3.setRadius(100);
+		System.out.println(c3);
+
```
Comment (leap92) [inline Topic 0/CircleClass/src/TestCircle.java]: Minor: extra spaces

File: Topic 0/CircleClass/src/TestCircle.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+public class TestCircle {
+
+	public static void main(String[] args) {
+
```
Comment (leap92) [inline Topic 0/CircleClass/src/TestCircle.java]: Minor: extra spaces

File: Topic 0/CircleClass/src/Circle.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		this.radius = radius;
+		this.color = "Red";
+	}
+	
+	public Circle (double radius, String color){
+		this.radius = radius;
+		this.color = color;
+	}
+
+
```
Comment (leap92) [inline Topic 0/CircleClass/src/Circle.java]: Minor: extra spaces

**Comment by Igrass97** [comment]
@leap92 I will remove the extra spaces and merge.

**Pull request merged**